bin/git/update-hooks

```diff
@@ -19,11 +56,15 @@ install_hook_wrapper() {
         # If a hook already exists and is a file:
         if [ -f $HOOK_DIR/$hook ]; then
             step_name "Moving existing '$hook' script to '$hook-local'"
-            mv $HOOK_DIR/$hook $HOOK_DIR/$hook-local
+            if [ $make_updates == 1 ]; then
```

Comment:
Both of these are just bash standards I'm used to, just for safety reasons. For example [[ builtin operator additionally provides compound commands and regular expression matching - like if [[ string =~ regex ]]. I doubt we'll add much bash to these scripts  - but I like to do it as suggested because sooner or later it gets copied and then as you've pointed out it introduces bugs. No biggy.